spec/features/user/user_logs_in_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-feature 'User Logs In' do
+feature 'User logs in' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.